app_dart/lib/src/request_handlers/refresh_chromebot_status.dart

```diff
@@ -119,14 +119,17 @@ class RefreshChromebotStatus extends ApiRequestHandler<Body> {
         final Task update = datastoreTask.task;
         update.status = latestLuciTask.status;
 
+        final CiYaml ciYaml = await scheduler.getCiYaml(datastoreTask.commit);
+        final Target target =
+            ciYaml.postsubmitTargets.singleWhere((Target target) => target.value.name == datastoreTask.task.name);
+
         /// Use `update.attempts - 1` as the `retries` to skip the initial run.
         if (await luciBuildService.checkRerunBuilder(
-            commit: datastoreTask.commit,
-            luciTask: latestLuciTask,
-            retries: update.attempts! - 1,
-            datastore: datastore,
-            repo: slug.name,
-            isFlaky: datastoreTask.task.isFlaky)) {
+          commit: datastoreTask.commit,
+          target: target,
+          task: update,
+          datastore: datastore,
```

Comment:
With `ignoreChecks` default false, this rerun will never be triggered.

Reply:
`ignoreChecks` is reserved for the reset-prod-test use case as humans likely have a better idea if we should retry (such as needing a 4th retry). In this case, refresh chromebot will trigger tests as long as it meets the conditions in checkRerunBuilder (< 3 attempts, latest datastore does not indicate it passed).